src/Component/spec/Metadata/Resource/Factory/AttributesResourceMetadataCollectionFactorySpec.php

```diff
@@ -335,4 +336,59 @@ function it_creates_resource_metadata_with_form_type(RegistryInterface $resource
         $operation->getRepository()->shouldReturn('app.repository.dummy');
         $operation->getFormType()->shouldReturn('App\Form\DummyType');
     }
+
+    function it_creates_resource_metadata_with_templates_dir(RegistryInterface $resourceRegistry): void
```

Comment:
for now, we are missing specs for config, where the template is null, and both templatedir and/or short name are null